sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/statsbeat/RpAttachType.java

```diff
@@ -9,7 +9,7 @@ public enum RpAttachType {
     STANDALONE_AUTO("StandaloneAuto"), // RP attach is enabled via a custom JAVA_OPTS or on premise resources
     INTEGRATED_AUTO("IntegratedAuto"); // RP attach is on by default
 
-    private static volatile RpAttachType attachType = MANUAL;
+    private static volatile RpAttachType attachType = STANDALONE_AUTO;
```

Comment:
do we need to change the default, or can we set it at agent startup?

Reply:
this is for customer who doesn't use RP integration and attach java agent via jvm arg. we could set it in the agtent startup.. is it necessary?

Reply:
it just seems like an odd default value to have in this repository, since this repository theoretically doesn't know about the Java agent repository

Reply:
java agent is using standalone exporter as a dependency. rpattachtype is here. we have lots of code specific for applicationinsights is here too.